recipes/nettrustscore/meta.yaml

```diff
@@ -0,0 +1,47 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "nettrustscore" %}
+{% set version = "1.0.1" %}
+{% set python_min = "3.7" %}
```

Comment:
Conda-forge does not officially support python <3.9 (which is the globally pinned version if you do not specify it in the recipe).